ecs/users/fields/custom/role.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: role
+  title: Wazuh's User Role
+  description: List of roles assigned to a user.
+  reusable:
+    top_level: true
```

Comment:
I don't think a role would be added at top level.

Reply:
Fixed on commit https://github.com/wazuh/wazuh-indexer-plugins/pull/303/commits/46c50892690e8669afab8fe1552f2b409a4af745